diff --git a/node_modules/nativewind/dist/metro/index.js b/node_modules/nativewind/dist/metro/index.js
index f638ba1..b053539 100644
--- a/node_modules/nativewind/dist/metro/index.js
+++ b/node_modules/nativewind/dist/metro/index.js
@@ -31,7 +31,7 @@ function withNativeWind(config, { inputPath, postcssPath } = {}) {
         cli.stdout.on("data", (data) => {
             const output = data.toString().trim();
             const createOptions = JSON.stringify((0, extract_1.getCreateOptions)(output));
-            (0, node_fs_1.writeFileSync)(outputFile, `const { NativeWindStyleSheet } from "nativewind/dist/style-sheet";\nNativeWindStyleSheet.create(${createOptions});`);
+            (0, node_fs_1.writeFileSync)(outputFile, `import { NativeWindStyleSheet } from "nativewind/dist/style-sheet";\nNativeWindStyleSheet.create(${createOptions});`);
         });
         cli.stderr.on("data", (data) => {
             const output = data.toString().trim();
